Testscripts/Windows/CPU-Offline.ps1

```diff
@@ -165,26 +176,45 @@ done
 SetTestStateCompleted
 "@
 			Set-Content "$LogDir\workCommand.sh" $workCommand
+		}
 
-			$setupcommand = @"
+		if ($isNetworkWorkloadEnable -eq 1) {
+			$targetIPAddress = $AllVMData[1].InternalIP
+			$workCommand = @"
+source utils.sh
+for jn in 1 2 3 4 5 6 7 8 9 10
+do
+iperf3 -c $targetIPAddress -t 60 -P 8 >> workload.json
```

Comment:
Standard_L32s_v2 has 32 vCPU, how do you like that we use 32 client streams to run, which is "-P 32" ?

Reply:
Changed